src/create_disk.sh

```diff
@@ -406,7 +406,7 @@ s390x)
     # this is only a temporary solution until we are able to do firstboot check at bootloader
     # stage on s390x, either through zipl->grub2-emu or zipl standalone.
     # See https://github.com/coreos/ignition-dracut/issues/84
-    # A similar hack is present in https://github.com/coreos/coreos-assembler/blob/main/src/gf-platformid#L55
+    # A similar hack is present in https://github.com/coreos/coreos-assembler/blob/main/src/gf-set-platform#L55
```

Comment:
Aside/optional: that line number will eventually become stale. We should probably either make it use a permalink, or just say `gf-set-platform`.